app/app.py

```diff
@@ -1,3 +1,5 @@
+print('git dojo')
+print('xD')
```

Comment:
não gostei do x minusculo
```suggestion
print('XD')
```